stripe-servant/src/Stripe/Api.hs

```diff
@@ -40,10 +42,20 @@ type PriceApi
   :<|> StripeAuth :> Capture ":product_id" PriceId :> Get '[JSON] Price
   :<|> StripeAuth :> QueryParam "lookup_keys[]" T.Text :> Get '[JSON] (StripeList Price)
 
-type SubscriptionApi
-  = StripeAuth :> ReqBody '[FormUrlEncoded] SubscriptionCreate :> Post '[JSON] Subscription
-  :<|> StripeAuth :> Capture ":subscription_id" SubscriptionId :> Get '[JSON] Subscription
-  :<|> StripeAuth :> QueryParam "customer" CustomerId :> Get '[JSON] (StripeList Subscription)
+type InvoiceApi =
+  StripeAuth :> Capture ":invoice_id" InvoiceId :> Get '[JSON] Invoice
+    :<|> StripeAuth :> QueryParam "customer" CustomerId :> QueryParam "expand[]" T.Text :> Get '[JSON] (StripeList Invoice)
```

Comment:
Would this work too?

```suggestion
    :<|> StripeAuth :> QueryParam "customer" CustomerId :> QueryParam "expand" [T.Text] :> Get '[JSON] (StripeList Invoice)
```

Reply:
GHCID complains when I try with your change